src/main/java/org/fogbowcloud/arrebol/api/http/controllers/JobController.java

```diff
@@ -27,21 +28,31 @@ public JobController(JobService jobService){
 
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<String> addJob(@RequestBody JobSpec jobSpec){
-        LOGGER.info("Adding new Job: " + jobSpec + ".");
+        LOGGER.info(String.format(Messages.Info.JobController.ADDING_NEW_JOB, jobSpec));
```

Comment:
I do not like this, why not add the string right here instead of using another class only to do that?